test/client-unit/services/StockService.spec.js

```diff
@@ -0,0 +1,126 @@
+      {
+        uuid : mockStockFormStoreData[0].lots[0].uuid,
+        label : mockStockFormStoreData[0].lots[0].lot,
+        initial_quantity : mockStockFormStoreData[0].lots[0].quantity,
+        quantity : mockStockFormStoreData[0].lots[0].quantity,
+        unit_cost : mockStockFormStoreData[0].unit_cost,
+        expiration_date : mockStockFormStoreData[0].lots[0].expiration_date,
+        inventory_uuid : mockStockFormStoreData[0].inventory_uuid,
+        origin_uuid : entityUuid,
+      },
+      {
+        uuid : mockStockFormStoreData[0].lots[1].uuid,
+        label : mockStockFormStoreData[0].lots[1].lot,
+        initial_quantity : mockStockFormStoreData[0].lots[1].quantity,
+        quantity : mockStockFormStoreData[0].lots[1].quantity,
+        unit_cost : mockStockFormStoreData[0].unit_cost,
+        expiration_date : mockStockFormStoreData[0].lots[1].expiration_date,
+        inventory_uuid : mockStockFormStoreData[0].inventory_uuid,
+        origin_uuid : entityUuid,
+      },
+    ];
+
+    const processed = Stock.processLotsFromStore(mockStockFormStoreData, entityUuid);
+    expect(processed).to.be.an('array');
+    expect(processed.length).to.be.equal(2);
+
+    const [first, second] = processed;
+    expect(first).to.be.an('object');
+    expect(second).to.be.an('object');
+    expect(processed).to.deep.equal(expectedLotsArray);
+  });
+
+  it('#uniformSelectedEntity() format the displayName property for a given entity object', () => {
+    const entityText = { text : 'Text supposed to be displayName' };
+    const entityName = { name : 'Name supposed to be displayName' };
+    const entityDisplayName = { display_name : 'Display_Name supposed to be displayName' };
+
+    const uniformText = Stock.uniformSelectedEntity(entityText);
+    const uniformName = Stock.uniformSelectedEntity(entityName);
+    const uniformDisplayName = Stock.uniformSelectedEntity(entityDisplayName);
+
+    expect(uniformText).to.have.property('displayName', entityText.text);
+    expect(uniformName).to.have.property('displayName', entityName.name);
+    expect(uniformDisplayName).to.have.property('displayName', entityDisplayName.display_name);
+  });
+
+  it('#getQueryString() returns a query string with parameters for a request', () => {
+    const FILTER_KEY_MOVEMENT = 'movement';
+    const FILE_TYPE_CSV = 'csv';
+    const query = Stock.getQueryString(FILTER_KEY_MOVEMENT, FILE_TYPE_CSV);
+    const params = query.split('&');
+    let fileTypeRendererExists = false;
+    let periodParamExists = false;
+    let limitParamExists = false;
+    for (let i = 0; i < params.length; i++) {
+      if (params[i] === `renderer=${FILE_TYPE_CSV}`) {
+        fileTypeRendererExists = true;
+      }
+      if (params[i].indexOf('period=') > -1) {
+        periodParamExists = true;
+      }
+      if (params[i].indexOf('limit=') > -1) {
+        limitParamExists = true;
+      }
+    }
+    expect(fileTypeRendererExists).to.be.equal(true);
```

Comment:
You can actually just use `chai` to do this - specifically the [include() method](http://www.chaijs.com/api/bdd/#method_include).  You would something like this work and save you from doing the `for` loop?

```js
expect(query).to.include(`renderer=${FILE_TYPE_CSV}`);
expect(query).to.include('period=');
expect(query).to.include('limit=');
```